src/main/java/uk/gov/ons/census/action/builders/PrintFileDtoBuilder.java

```diff
@@ -17,7 +17,10 @@
           ActionType.P_RL_2RL1A,
           ActionType.P_RL_2RL2BA,
           ActionType.P_RL_1IRL1,
-          ActionType.P_RL_1IRL2B
+          ActionType.P_RL_1IRL2B,
+          ActionType.P_RL_1RL1B,
+          ActionType.P_RL_1RL2BB,
+          ActionType.P_RL_1RL4A
```

Comment:
This needs to be `P_RL_1RL4` not `P_RL_1RL4A` because it will be sent to people who have already launched EQ, not people who haven't (who have potentially misplaced their UAC)

Reply:
I've changed and pushed so that this list now has 4 rather than 4A.

Reply:
I have added codes `P_RL_2RL1` and `P_RL_2RL2B` and have run the maven check and format goals.